plothist/__init__.py

```diff
@@ -54,3 +54,17 @@
 
 with resources_path("plothist", "default_style.mplstyle") as style_file:
     plt.style.use(style_file.as_posix())
+
+# Check the fonts
+from matplotlib.font_manager import findfont
+
+print_font_message = False
+for font_type in ["Math", "Sans", "Roman"]:
+    try:
+        findfont(f"Latin Modern {font_type}", fallback_to_default=False)
+    except:
+        print_font_message = True
+if print_font_message:
+    print(
```

Comment:
can be put in the `except` block. Also, this should not be a print, but a warning:
```
import warnings
warnings.warn("Warning...........Message")
```